---
title: '&#39;Exit t r y&#39; 안에서 나타날 수는 &#39;시도&#39; 문'
ms.date: 07/20/2015
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords:
- BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
ms.openlocfilehash: a33f5750a0ba8f833e623101467327635bf0c8d8
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 05/04/2018
ms.locfileid: "33614484"
---
# <a name="39exit-try39-can-only-appear-inside-a-39try39-statement"></a>&#39;Exit t r y&#39; 안에서 나타날 수는 &#39;시도&#39; 문
`Exit Try` 는 `Try` 블록 문에서만 나올 수 있습니다. 중복 `Exit Try` 문이 있거나 또는 `Exit Try` 문이 해당 `Try` 블록의 경계를 벗어난 곳에 나타납니다.  
  
 **오류 ID:** BC30393  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
1.  불필요한 `Exit Try` 문을 찾아서 제거합니다.  
  
2.  이동 된 `Exit Try` 문을 코드 내에서 적절 한 위치로 합니다.  
  
## <a name="see-also"></a>참고 항목  
 [Try...Catch...Finally 문](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
